src/cmd/factory/impl/onedrive.go

```diff
@@ -35,20 +36,23 @@ func handleOneDriveFileFactory(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	gc, acct, inp, err := getGCAndVerifyUser(ctx, User)
+	gc, acct, inp, err := getGCAndVerifyResourceOwner(ctx, connector.Users, User)
 	if err != nil {
 		return Only(ctx, err)
 	}
 
-	deets, err := generateAndRestoreOnedriveItems(
+	sel := selectors.NewOneDriveBackup([]string{User}).Selector
```

Comment:
Should this be a restore selector?

```suggestion
	sel := selectors.NewOneDriveRestore([]string{User}).Selector
```

Reply:
technically doesn't matter, since we're only using it as an IDNameProvider.  I think I wrote this before the provider mock existed.  I'll switch it to use that instead of the selector.